store/sqlstore/user_store.go

```diff
@@ -592,9 +596,30 @@ func (us SqlUserStore) GetAllProfilesInChannel(channelId string, allowFromCache
 	if err != nil {
 		return nil, model.NewAppError("SqlUserStore.GetAllProfilesInChannel", "store.sql_user.app_error", nil, err.Error(), http.StatusInternalServerError)
 	}
-
 	var users []*model.User
-	if _, err := us.GetReplica().Select(&users, queryString, args...); err != nil {
+	rows, err := us.GetReplica().Db.Query(queryString, args...)
+	if err != nil {
+		return nil, model.NewAppError("SqlUserStore.GetAllProfilesInChannel", "store.sql_user.get_profiles.app_error", nil, err.Error(), http.StatusInternalServerError)
+	}
+
+	defer rows.Close()
+	for rows.Next() {
+		var user model.User
+		var propsBytes []uint8
```

Comment:
`uint8` is slightly out of place here. Let's just go with `[]byte`. And also I think it is more readable if we just drop the "Bytes" suffix from the end.

I would suggest something like: `var props, notifyProps, timezone []byte`